cke-tools/src/README.md

```diff
@@ -18,3 +19,27 @@ CKE tools is licensed under MIT license.
 
 [godoc]: https://godoc.org/github.com/cybozu/neco-containers/cke-tools/src
 [CKE]: https://github.com/cybozu-go/cke
+
+`updateblock117`
+----------------
+
+`updateblock117` is the one-shot program used by CKE when upgrading Kubelet from 1.16 to 1.17.
+
+`updateblock117` has two subcommands.
+
+### `updateblock117 need-update <block-pv-name>`
+
+Check that we should modify the path of the target device file or not.
+
+`block-pv-name` is the name of the PersistentVolume object.
+
+- Returns stdout `{"result": "yes"}` if the device needs to tweak.
```

Comment:
> - Returns stdout `{"result": "yes"}` if the device needs to tweak.
> - Returns stdout `{"result": "no"}` if the device needs to tweak.

Probably one is wrong. Note that `if the device needs to tweak.` appears twice.
Also, the text sounds unnatural.  `when the device file needs to be relocated` is better.